templates/newrole.html

```diff
@@ -0,0 +1,98 @@
+{{ define "content" }}
+  <form method="post" action="/roles">
+    <!-- TODO: csrf -->
+    <label class="block">
+      <span class="form-label">Name</span>
+      <span class="align-top text-sm text-gray-500">*</span>
+      <!-- TODO: inline errors -->
+      {{ if .name_err }}
+        {{ range .name_err }}
+          <span class="form-error">{{ . }}</span>
+        {{ end }}
+      {{ end }}
+      <input name="name" class="form-input mb-3"  value="" required>
+    </label>
+    <label class="block">
+      <span class="form-label">Email</span>
+      <span class="align-top text-sm text-gray-500">*</span>
+      {{ if .email_err }}
+        {{ range .email_err }}
+          <span class="form-error">{{ . }}</span>
+        {{ end }}
+      {{ end }}
+      <input name="email" class="form-input mb-3" value="" required>
+    </label>
+    <label class="block">
+      <span class="form-label">Phone</span>
+      {{ if .phone_err }}
+        {{ range .phone_err }}
+          <span class="form-error">{{ . }}</span>
+        {{ end }}
+      {{ end }}
+      <input type="phone" name="phone" class="form-input mb-3" value="">
```

Comment:
Could be good to call out that this gets displayed publicly on the view page. I suppose email as well.